neuralop/layers/local_fno_block.py

```diff
@@ -389,7 +466,7 @@ def forward_with_preactivation(self, x, index=0, output_shape=None):
 
         x_fno = self.convs[index](x, output_shape=output_shape)
 
-        x = x_fno + x_skip_fno_diff
+        x = x_fno + x_skip_fno_diff_disco
```

Comment:
We could potentially just call this `x_skip_local_fno`

Reply:
That sounds good. Changing